frontend/scripts/react-components/tool/sankey/node-menu.component.jsx

```diff
@@ -1,12 +1,21 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 import Text from 'react-components/shared/text';
 
 import './node-menu.scss';
 
+const ITEM_HEIGHT = 33;
+
 function NodeMenu(props) {
-  const { options, menuPos, isVisible } = props;
+  const { options, menuPos, isVisible, containerRef } = props;
+  const menuHeight = useMemo(() => ITEM_HEIGHT * options?.length, [options]);
+  const flipped = useMemo(() => {
```

Comment:
I would drop memo, because everytime the calculations need to be done the memo function will be called. There's no advantage here.